tests/queryset_pickle/models.py

```diff
@@ -57,3 +57,24 @@ class SomeModel(models.Model):
 
 class M2MModel(models.Model):
     groups = models.ManyToManyField(Group)
+
+
+class QueueItem(models.Model):
```

Comment:
As in `model_regress/test_pickle.py`, you can probably put the model in the test and test with an in-memory instance rather than using `.create()`.